src/meltano/cli/config.py

```diff
@@ -131,6 +131,12 @@ def get_label(metadata) -> str:
     default="json",
 )
 @click.option("--extras", is_flag=True, help="View or list only plugin extras.")
+@click.option(
+    "--safe/--unsafe",
```

Comment:
Love this! Can we get a quick update to https://docs.meltano.com/reference/command-line-interface#config as well?

Reply:
Docs look great. Thanks @ReubenFrankel!